### DISCUSSION TITLE
Weekly Release Cycles

### DISCUSSION THREAD

**Discussion by compgeniuses** [ellite/Wallos#454] [General]
I have noted with concern that most of the releases for wallos seem to come any time, any moment and any day.

Wouldnt it be great to have major releases scheduled weekely, with minor and bug fixes released when noted?

**Comment by ellite** [comment]
Hi. Thank you for your suggestion.
I don't have much free time to work on Wallos, with a full time job, a baby and life in general. So I can't commit to a schedule.
Sorry, new versions will keep being released as I have availability to develop them.
